modules/ROOT/pages/access-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+= Access Recovery
+
+If you've lost the private key of an SSH keypair used to log into Fedora CoreOS, and do not have any password logins set up to use at the console, you can gain access back to the machine using the `init=/bin/sh` trick:
+
+. Launch the system using `init=/bin/sh`
```

Comment:
Needs further explanation. (I understand, but it's best to be as clear as possible.)

Reply:
Procedure should also be linked from this page:
 https://docs.fedoraproject.org/en-US/fedora-coreos/migrate-cl/ 

e.g.
```
The coreos.autologin kernel command-line parameter is not currently supported in FCOS.
See modules/ROOT/pages/access-recovery.adoc for access recovery procedure.
```

Reply:
Good point, will tweak.

Reply:
Done below! :arrow_down: